src/main/java/com/hyperwallet/clientsdk/model/HyperwalletBankAccount.java

```diff
@@ -21,6 +21,7 @@
 
     private Type type;
     private Status status;
+    private Status transition;
```

Comment:
Bank Account transition has only two possible states:
1. DE_ACTIVATED
2. ACTIVATED
Please create a separate enum to fix this.

Reply:
Per the documentation possible status' are as given below.

ACTIVATED
VERIFIED
INVALID
DE_ACTIVATED

and for transitions they are:

ACTIVATED
VERIFIED
INVALID
DE_ACTIVATED

They are declared in the following lines as enumerations already. Status is used for transition too.
    public static enum Status {ACTIVATED, VERIFIED, INVALID, DE_ACTIVATED}

Let me know if anymore change is still required.

Reply:
I'm referring to this link: https://docs.hyperwallet.com/content/api/v3/resources/status-transitions/bank-accounts-create for Bank Account Status transition. Please let me know if I have to refer to another link.

Reply:
This is an inconsistency. A new ticket has to be created for this to update the document. However, addition of Verified and Invalid in the query will not break the functionality. Please approve the request now as I am going by the document https://docs.hyperwallet.com/content/api/v4/resources/status-transitions/bank-accounts-retrieve for now.